source/drivers/driver-compatibility-reference.txt

```diff
@@ -718,6 +786,14 @@ Language Compatibility
      - PHP 7.0
      - HHVM 3.9
 
+   * - mongodb-1.2
+     -
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
```

Comment:
PHP 7.1's release date is December 1 (see: [here](https://wiki.php.net/todo/php71)), so you might as well add a column for that.

@derickr can chime in if additional HHVM versions should be listed. I believe 3.15 is the most recent LTS (3.9 being the last), and that is supported by the current HHVM driver.